public/app/features/data-connections/constants.ts

```diff
@@ -0,0 +1,12 @@
+// The ID of the app plugin that we render under that "Cloud Integrations" tab
+export const CLOUD_ONBOARDING_APP_ID = 'grafana-easystart-app';
+
+// The ID of the main nav-tree item (the main item in the NavIndex)
+export const ROUTE_BASE_ID = 'data-connections';
+
+export enum ROUTES {
```

Comment:
Should this live in `types.ts`?

Reply:
Thanks mate, good point, those enums should live in the same place. 👍 

I just realised thought that the `types.ts` file is actually not referenced by any module, so will get rid of it for now.